packages/angular_devkit/build_angular/src/angular-cli-files/models/build-options.ts

```diff
@@ -15,6 +15,7 @@ import {
   Budget,
   CurrentFileReplacement,
   ExtraEntryPoint,
+  SourceMap,
```

Comment:
Name this `SourceMapOption` or something. `SourceMap` is also a node module we're using.